### DISCUSSION TITLE
Docker Healthcheck

### DISCUSSION THREAD

**Discussion by Digi92** [mosparo/mosparo#260] [Ideas]
Hello, 
I wanted to suggest adding a health check to the Docker container to ensure the service is running properly.
See: [https://docs.docker.com/reference/dockerfile/#healthcheck](https://docs.docker.com/reference/dockerfile/#healthcheck)

As an additional suggestion, I have created a ‘health’ route as a small controller action for external monitoring.
```
namespace Mosparo\Controller;

use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\HttpFoundation\JsonResponse;
use Symfony\Component\Routing\Annotation\Route;

class HealthController extends AbstractController
{
    #[Route('/health', name: 'app_health')]
    public function indexAction()
    {
        return new JsonResponse(array(
            'service' => 'captcha',
            'healthy' => true,
        ));
    }
}
```

What do you think?
I think this would be a useful addition for monitoring the health of the Mosparo service.

**Comment by zepich** [comment]
Hi @Digi92 

Thank you very much for your suggestion. 

That's an excellent idea! I will add such a controller to mosparo.

Out of curiosity: Did you have any troubles with the mosparo container regarding health?

I wish you a nice day.

Kind regards,

zepich

**Reply by Digi92** [reply]
Hi @zepich,
so far i have had no problems with health.

I hope you have a great day too.

**Comment by zepich** [comment]
Added with v1.3
